AdtSampleApp/SampleClientApp/CommandLoop.cs

```diff
@@ -121,13 +120,18 @@ public async Task CommandGetModels(string[] cmd)
             Log.Alert($"Submitting...");
             try
             {
-                var reslist = new List<ModelData>();
-                AsyncPageable<ModelData> results = client.GetModelsAsync(dependenciesFor, includeModelDefinitions);
-                await foreach (ModelData md in results)
+                AsyncPageable<DigitalTwinsModelData> results = client.GetModelsAsync(
+                    new GetModelsOptions
+                    {
+                        DependenciesFor = dependenciesFor,
+                        IncludeModelDefinition = includeModelDefinitions
+                    });
+                var reslist = new List<DigitalTwinsModelData>();
+                await foreach (DigitalTwinsModelData md in results)
                 {
                     Log.Out(md.Id);
```

Comment:
add more description here?

Retrieved model '{md.Id}'

Reply:
As discussed, just updating to work with the new nuget.